util/unifyfs/src/unifyfs-rm.c

```diff
@@ -539,6 +657,45 @@ static size_t construct_server_argv(unifyfs_args_t* args,
     return argc;
 }
 
+/**
+ * @brief Default data stage launch routine
+ *
+ * @param args        The command-line options
+ * @param stage_args  unifyfs-stage argument vector to be filled
+ *
+ * @return number of server arguments
+ */
+static size_t construct_stage_argv(unifyfs_args_t* args,
```

Comment:
Returning a `size_t` from this is weird.  `size_t` is only used for returning the size of an object.  Use an unsigned int.

Reply:
The new function construct_stage_argv uses the same overall logic flow as the function construct_server_argv already existing in the current dev branch, which also returns size_t.  

If you'd like to overhaul how the functions in unifyfs-rm are written and/or structured, by all means let's open a new issue to overhaul that file, but let's do it as a separately from adding stage-in/stage-out functionality, which we've written to mirror the existing function templates in that file.